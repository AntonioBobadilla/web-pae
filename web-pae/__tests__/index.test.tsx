import Home from '@/pages/student/home';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';

describe('Home', () => {
  it('renders homepage unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
