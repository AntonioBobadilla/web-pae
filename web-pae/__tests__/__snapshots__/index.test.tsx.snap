// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders index page unchanged 1`] = `
<div>
  <div>
    <div
      class="page"
    >
      <div
        class="header"
      >
        <div
          class="logoContainer"
        >
          <img
            class="logo"
            src="/images/logo.png"
          />
        </div>
        <div
          class="sections"
        >
          <a
            class="section"
            href="#qs"
          >
             
            Quiénes somos
             
          </a>
          <a
            class="section"
            href="#cf"
          >
             
            Cómo funciona
             
          </a>
          <a
            class="section"
            href="#ubi"
          >
             
            Ubicación
             
          </a>
        </div>
        <div
          class="buttons"
        >
          <div
            class="btn"
          >
            <div
              class="btn"
            >
              <button
                class="login"
              >
                Ingresa
              </button>
              <button
                class="signup"
              >
                Regístrate
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="imageSection"
      >
        <img
          class="mainImage"
          src="/images/landing.jpeg"
        />
        <span
          class="mainText"
        >
          <br />
          PROGRAMA 
          <br />
          ASESOR 
          <br />
          ESTUDIANTE
          <br />
        </span>
      </div>
      <div
        class="body"
      >
        <span
          class="title"
          id="qs"
        >
          ¿Quiénes somos?
        </span>
        <hr
          class="hr"
        />
        <span
          class="description"
        >
          En el Programa Asesor Estudiante (PAE) recibirás
           
          <strong
            class="bold"
          >
             asesorías personalizadas 
          </strong>
           
          con el apoyo de un alumno quien además de tener un horario similar al tuyo, también tiene la mejor intención de ayudarte a
           
          <strong
            class="bold"
          >
            resolver todas tus dudas
          </strong>
          .
        </span>
        <span
          class="title"
          id="cf"
        >
          ¿Cómo funciona?
        </span>
        <hr
          class="hr"
        />
        <span
          class="description"
        >
          Tú seleccionas (con 24 horas de anticipación) el
           
          <strong
            class="bold"
          >
            horario
          </strong>
           en el que te gustaría recibir la asesoría, la
           
          <strong
            class="bold"
          >
            materia 
          </strong>
          y los
           
          <strong
            class="bold"
          >
            temas o dudas específicas 
          </strong>
          en los que necesites apoyo; posteriormente, recibirás un correo de confirmación con información importante sobre la realización de tu asesoría.
        </span>
        <span
          class="title"
          id="ubi"
        >
          Ubicación
        </span>
        <hr
          class="hr"
        />
        <div
          class="location"
        >
          <div
            class="locationL"
          >
            <div
              class="locationT"
            >
              <span
                class="locationPAE"
              >
                <br />
                <strong
                  class="bold"
                >
                   P
                </strong>
                ROGRAMA 
                <br />
                <strong
                  class="bold"
                >
                   A
                </strong>
                SESOR 
                <br />
                <strong
                  class="bold"
                >
                   E
                </strong>
                STUDIANTE
                <br />
              </span>
              <img
                class="bookImage"
                src="/images/pae-logo.png"
              />
            </div>
            <div
              class="locationB"
            >
              <span
                class="descriptionLocation"
              >
                Encuéntranos en Aulas1 
                <br />
                <strong
                  class="bold"
                >
                  A1234
                </strong>
                <br />
              </span>
              <span
                class="descriptionLocation"
              >
                Horario de atención: 
                <br />
                <strong
                  class="bold"
                >
                  8:00am - 5:00pm
                </strong>
                <br />
              </span>
              <div
                class="linkCont"
              >
                <a
                  class="link"
                  href="https://www.google.com"
                >
                  Solicita tu asesoría
                </a>
              </div>
            </div>
          </div>
          <div
            class="locationR"
          >
            <img
              class="imageLocation"
              src="/images/office.jpg"
            />
          </div>
        </div>
        <span
          class="greetings"
        >
          ¡En PAE estamos encantados de poder ayudarte!
           
        </span>
      </div>
      <div
        class="credits"
      >
        <div
          class="legal"
        >
          <span
            class="creditsTitle"
          >
             Legal 
          </span>
          <span
            class="creditsTitle"
          >
             Legal 
          </span>
          <span
            class="creditsTitle"
          >
             Legal 
          </span>
        </div>
        <div
          class="logoHacket"
        >
          <img
            class="hacketImg"
            src="/images/hacket.png"
          />
        </div>
        <div
          class="credit"
        >
          Credits
        </div>
      </div>
      <div
        class="copyrights"
      >
        <span>
          Registrada ® 
        </span>
      </div>
    </div>
  </div>
</div>
`;
